frontend/src/components/ideas/IdeasBoard.js

```diff
@@ -93,10 +93,29 @@ export default function IdeasBoard({
   };
 
   const handleRemoveComment = (c) => {
```

Comment:
It is not best practice to use alphabet characters like this for argument name. Use a readable argument name. I would recommend updating this method.

Reply:
Renamed arguments and variables.